plugins/shopify/src/editors/ShopifyCollectionPicker.tsx

```diff
@@ -204,7 +204,7 @@ export class ShopifyCollectionPicker extends SafeComponent<ShopifyCollectionPick
     return {
       '@type': '@builder.io/core:Request',
       request: {
-        url: `${appState.config.apiRoot()}/api/v1/shopify/collections/{{this.options.collection}}.json?apiKey=${
+        url: `${appState.config.apiRoot()}/api/v1/shopify/collections/${collectionId}.json?apiKey=${
```

Comment:
If we're interpolating directly, I think we can get rid of the `options` object below, right? Or is it used for something else?

https://github.com/BuilderIO/builder/blob/3bf85e43c5dc9d307267d993f2e1ff5feb3845b5/plugins/shopify/src/editors/ShopifyCollectionPicker.tsx#L211-L213